pdns/test-ueberbackend_cc.cc

```diff
@@ -362,8 +369,11 @@ static void testWithoutThenWithCache(std::function<void(UeberBackend& ub)> func)
     ::arg().set("query-cache-ttl")="20";
     ::arg().set("negquery-cache-ttl")="60";
     QC.cleanup();
+    ::arg().set("domain-cache-ttl")="60";
```

Comment:
I would not mind adding a separate test here. We currently have a first test with no cache at all and a second one with the positive and negative query caches. This PR adds the domain cache to the second test, I'd rather keep the second test without the domain cache and have a third one with the positive, negative and domain caches.